sql/core/src/test/resources/sql-tests/inputs/udf/pgSQL/udf-aggregates_part1.sql

```diff
@@ -0,0 +1,156 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- AGGREGATES [Part 1]
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/aggregates.sql#L1-L143
```

Comment:
Can we use `REL_12_BETA2` instead of `REL_12_BETA1`? It will be the same (I guess).

Reply:
I will make a separate PR to fix everywhere right away.